lib/common/components/EC2InstanceCard.js

```diff
@@ -106,6 +106,9 @@ export default class EC2InstanceCard extends Component<Props> {
             <div>
               <Icon type='rocket' /> {formatTimestamp(instance.launchTime)}
             </div>
+            <div>
+              <Icon type='balance-scale' /> {instance.instanceType}
```

Comment:
I think `server` would be more appropriate here.

https://fontawesome.com/v4.7.0/icon/server